mllm/backends/ascend/AscendCommon.cpp

```
@@ -0,0 +1,187 @@
+// Copyright (c) MLLM Team.
+// Licensed under the MIT License.
+
+#include "mllm/backends/ascend/AscendCommon.hpp"
+
+#include <vector>
+
+#include "mllm/backends/ascend/memory/AscendMemoryManager.hpp"
+
+namespace mllm::ascend {
+
+static aclrtStream g_atb_stream = nullptr;
```

Clang (14.0.6)

[error] variable 'g_atb_stream' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables,-warnings-as-errors)